src/bulk_transfer.py

```diff
@@ -0,0 +1,42 @@
+import csv
+import bittensor
+
+# Initialize the wallet and subtensor
+wallet = bittensor.wallet(name="miner")
+subtensor = bittensor.subtensor("ws://100.28.51.29:9945")
+
+# Read the CSV file and extract Coldkeys
+coldkeys = []
+with open("tTAO requests.csv", mode='r') as file:
```

Comment:
nitpick: if file is not there does it fail gracefully (not much python savvy myself 😅)?